homeassistant/components/zwave_js/update.py

```diff
@@ -179,8 +187,23 @@ async def async_added_to_hass(self) -> None:
             )
         )
 
+        if self.hass.state == CoreState.running:
```

Comment:
We can use `async_at_start` helper.

https://github.com/home-assistant/core/blob/852b0caf5be4bba0dcaaf5f6a38221d1590c4ed9/homeassistant/helpers/start.py#L11-L19